eventstore/whatsapp_actions.py

```diff
@@ -26,3 +26,20 @@ def handle_operator_message(message):
             flow=settings.RAPIDPRO_OPERATOR_REPLY_FLOW, urns=[f"tel:{msisdn}"]
         ),
     ).delay(whatsapp_contact_id, message.id)
+
+
+def handle_inbound(message):
+    """
+    Triggers all the actions that are required for this inbound message
+    """
+    if not message.fallback_channel:
+        update_rapidpro_preferred_channel(message)
+
+
+def update_rapidpro_preferred_channel(message):
+    whatsapp_contact_id = message.data["_vnd"]["v1"]["chat"]["owner"]
```

Comment:
Is Turn always guaranteed to have this key in inbounds? I know we have a bunch of inbound messages in our eventstore without this, but that might've been a bug that has since been fixed.

Reply:
Oh thanks, I'll need to confirm this.